docs/api-reference/v3-sdk.md

```diff
@@ -1,7 +1,7 @@
 import FeedbackComponent from "@site/src/pages/feedback.md";
 
-# SDK v5.x.x to v3.x.x
+# SDK 3.x.x to v5.x.x
 
-The SDK v5.x.x to v3.x.x API docs can be viewed [here](https://lit-js-sdk-v3-api-docs.vercel.app/).
+The SDK v3.x.x to v5.x.x API docs can be viewed [here](https://lit-js-sdk-v3-api-docs.vercel.app/). Please note that the latest API docs are v6.x.x.
```

Comment:
We should add `Please note that the latest API docs are v6.x.x.` as:

```
:::warning
Please note that the latest API docs are v6.x.x.
:::
```

Reply:
Made the warning